i18n-helpers/src/bin/mdbook-xgettext.rs

```diff
@@ -111,12 +259,34 @@ fn main() -> anyhow::Result<()> {
         .ok_or_else(|| anyhow!("Missing output.xgettext.pot-file config value"))?
         .as_str()
         .ok_or_else(|| anyhow!("Expected a string for output.xgettext.pot-file"))?;
+    // The depth on which to split the output file. 0 (default) is to include
```

Comment:
Zero is not actually the default value, I get an error if I run this without a `depth` field :slightly_smiling_face: 

You will need something a bit more complex, something along the lines of

```rust
let depth = match cfg.get("depth") {
  None => 0,
  Some(depth) => depth.as_integer().with_context("Expected a number for output.xgettext.depth"))?,
};
```